pom.xml

```diff
@@ -1025,17 +1045,17 @@
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-lang3</artifactId>
-        <version>3.5</version>
+        <version>3.7</version>
       </dependency>
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-text</artifactId>
-        <version>1.1</version>
+        <version>1.2</version>
```

Comment:
i remember this one should be the same as in caps project and 1.1 was chosen exactly for that reason not so long time ago